prow/cmd/deck/main.go

```diff
@@ -308,20 +309,24 @@ func main() {
 
 	// cookie secret will be used for CSRF protection and should be exactly 32 bytes
 	// we sometimes accept different lengths to stay backwards compatible
-	var csrfToken []byte
+	var csrfToken [32]byte
 	if o.cookieSecretFile != "" {
 		cookieSecretRaw, err := loadToken(o.cookieSecretFile)
 		if err != nil {
 			logrus.WithError(err).Fatal("Could not read cookie secret file")
 		}
 		decodedSecret, err := base64.StdEncoding.DecodeString(string(cookieSecretRaw))
-		csrfToken = decodedSecret
 		if err != nil {
 			logrus.WithError(err).Fatal("Error decoding cookie secret")
 		}
+		if len(decodedSecret) == 32 {
```

Comment:
I couldn't find a nicer way to do this without it complaining about the array length